packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -76,7 +76,31 @@ class PageController extends ScrollController {
   final double viewportFraction;
 
   /// The current page displayed in the controlled [PageView].
+  ///
+  /// There are circumstances that this [PageController] can't know the current
+  /// page. Mainly when no [PageView]s are attached. Reading [page] will throw
```

Comment:
We should just be explicit:

`When a [PageController] has not yet been used by a [PageView], or when it is being used by multiple [PageView]s,
it cannot know the current page. ...`

Reply:
or maybe just remove the second sentence here since it's redundant with the list?

Reply:
Done